sympy/plotting/series.py

```diff
@@ -1,23 +1,77 @@
 ### The base class for all series
 from collections.abc import Callable
+from sympy.calculus.util import continuous_domain
+from sympy.concrete import Sum, Product
 from sympy.core.containers import Tuple
+from sympy.core.expr import Expr
 from sympy.core.function import arity
+from sympy.core.symbol import Symbol
+from sympy.functions import atan2, zeta, frac, ceiling, floor, im
 from sympy.core.relational import (Equality, GreaterThan, StrictGreaterThan,
     LessThan, StrictLessThan)
 from sympy.core.sympify import sympify
 from sympy.external import import_module
+from sympy.plotting.utils import _get_free_symbols, extract_solution
+from sympy.sets.sets import Set, Interval, Union
+from sympy.simplify.simplify import nsimplify
 from sympy.utilities.exceptions import sympy_deprecation_warning
-from .experimental_lambdify import (vectorized_lambdify, lambdify,
+from .experimental_lambdify import (vectorized_lambdify,
+    #lambdify,
     experimental_lambdify)
+from sympy.utilities.lambdify import lambdify
 from .intervalmath import interval
 import warnings
 
 
+def _uniform_eval(f1, f2, *args, modules=None,
+    force_real_eval=False, has_sum=False):
+    """
+    Note: this is an experimental function, as such it is prone to changes.
+    Please, do not use it in your code.
+    """
+    np = import_module('numpy')
+
+    def wrapper_func(func, *args):
+        try:
+            return complex(func(*args))
+        except (ZeroDivisionError, OverflowError):
+            return complex(np.nan, np.nan)
+
+    # NOTE: np.vectorize is much slower than numpy vectorized operations.
+    # However, this modules must be able to evaluate functions also with
+    # mpmath or sympy.
+    wrapper_func = np.vectorize(wrapper_func, otypes=[complex])
+
+    skip_fast_eval = False
+    if modules == "sympy":
+        skip_fast_eval = True
+
+    try:
+        if skip_fast_eval:
+            raise Exception
```

Comment:
I think that you should fix the test failing about exception. The tests are not running because of this, so I can't review the integration easily.